src/app/components/PortraitVideoCarousel/PortraitVideoCarouselNavigation/index.test.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { RefObject } from 'react';
+import {
+  act,
+  screen,
+  render,
+  fireEvent,
+} from '#app/components/react-testing-library-with-providers';
+import PortraitCarouselNavigation from '.';
+import { PROMO_ITEM_WIDTH } from '..';
+
+jest.useFakeTimers();
+
+const createMockScrollElementRef = ({
+  startScrollPosition,
+  itemCount,
+  paneWidth,
+}: {
+  startScrollPosition: number;
+  itemCount: number;
+  paneWidth: number;
+}) => {
+  const mockHTMLElementRef = {
+    current: {
+      scrollLeft: startScrollPosition,
+      scrollWidth: PROMO_ITEM_WIDTH * itemCount,
+      clientWidth: PROMO_ITEM_WIDTH * paneWidth,
+      scrollBy: jest.fn(),
+      addEventListener: jest.fn(),
+      removeEventListener: jest.fn(),
+    },
+  } as unknown as RefObject<HTMLDivElement>;
```

Comment:
I think `as unknown` temporarily erases the type by treating the type as `any` - wondering if theres an alternative type setup here that might be stronger for the tests?